content/blog/2025/custom-functions.md

```diff
@@ -0,0 +1,185 @@
+---
+title: Custom CSS Functions in the Browser
+sub: Start using author-defined functions
+author: miriam
+date: 2025-03-13
```

Comment:
I don't think there's much reason to backdate this. We may as well use the date we're publishing here.

```suggestion
date: 2025-04-11
```